.github/workflows/release.yml

```diff
@@ -28,6 +28,7 @@ jobs:
           helm repo add charts-incubator https://charts.helm.sh/incubator
           helm repo add charts-stable https://charts.helm.sh/stable
           helm repo add cas-postgres https://bcgov.github.io/cas-postgres/
+          helm repo add cas-postgres https://bcgov.github.io/cas-airflow/
```

Comment:
this cas-postgres should be renamed to cas-airflow?

Reply:
correct